src/protocolsupport/protocol/typeremapper/id/IdRemapper.java

```diff
@@ -282,10 +282,37 @@ protected HashMapBasedIdRemappingTable createTable() {
 		}
 	};
 
+	public static final IdRemappingRegistry<ArrayBasedIdRemappingTable> WINDOWTYPE = new IdRemappingRegistry<ArrayBasedIdRemappingTable>() {
```

Comment:
Move to pe specific typeremapper package

Reply:
Ok

Reply:
Lol it seems it's already there. So this was a duplicate.